src/cli_util.rs

```diff
@@ -230,6 +230,9 @@ jj init --git-repo=.";
                     repo_dir.to_str().unwrap()
                 )));
             }
+            Err(WorkspaceLoadError::Io(e)) => {
+                return Err(CommandError::InternalError(e.to_string()));
```

Comment:
Should this be `UserError`?

Reply:
The cases where I imagine this triggering are actually permissions errors and full filesystems, which aren't necessarily repo corruption, and which we certainly don't want bug reports for. On that basis I think I'll switch to `UserError`.